lib/spack/spack/test/data/targets/linux-centos7-power8

```diff
@@ -0,0 +1,645 @@
+processor	: 0
+cpu		: POWER8NVL (raw), altivec supported
+clock		: 2061.000000MHz
+revision	: 1.0 (pvr 004c 0100)
+
+processor	: 1
```

Comment:
Make this and other examples only contain one entry from the real `/proc/cpuinfo` -- I do not think we need more than that.

Reply:
Done